modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -257,13 +266,17 @@ public CacheMetricsImpl(GridCacheContext<?, ?> cctx, boolean isNear) {
         if (cctx.isNear())
             dhtCtx = cctx.near().dht().context();
 
-        if (cctx.store().store() instanceof GridCacheWriteBehindStore)
-            store = (GridCacheWriteBehindStore)cctx.store().store();
+            if (cctx.store() instanceof GridCacheWriteBehindStore)
```

Comment:
Is it a correct change?

If `cctx.store()` is an instance of `GridCacheWriteBehindStore` it doesn' mean that `cctx.store().store()` is a `GridCacheWriteBehindStore`.

Also it looks like added indentation is redundant

Reply:
@solveme Thanks for pointing it out. The change was made unintentional, when I was battling with intellij which enforced many formatting changes, I might have accidentally removed a `.store()`. I have restored it, also all the indentations enforced by intellij are restored to their original states. will be included in the next commit.